.github/workflows/release-bumper.yml

```diff
@@ -11,24 +11,38 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: master
+
       - uses: actions/setup-go@v2
         with:
           go-version: '^1.15' # The Go version to download (if necessary) and use.
+
       - name: Check for new releases and update
-        run: ./automation/release-bumper/release-bumper.sh
-      - name: Set "UPDATED_COMPONENT" environment variable
         run: |
-          echo 'UPDATED_COMPONENT<<EOF' >> $GITHUB_ENV
-          cat updated_component.txt >> $GITHUB_ENV
-          echo 'EOF' >> $GITHUB_ENV
-      - name: Set "UPDATED_VERSION" environment variable
-        run: |
-          echo 'UPDATED_VERSION<<EOF' >> $GITHUB_ENV
-          cat updated_version.txt >> $GITHUB_ENV
-          echo 'EOF' >> $GITHUB_ENV
+          ./automation/release-bumper/release-bumper.sh
+          if [[ -f updated_component.txt ]]; then
```

Comment:
why can't you use the github actions built in `if` command (as in line 8) instead of integrating it into an ad-hoc shell command?

Reply:
ok, disregard. the `if` is for skipping the entire step.